chapters/hub/build-pod-image.md

```diff
@@ -1,73 +1,93 @@
+- Your Executor requires certain Linux headers that can only be installed via `apt` or `yum`, but you don't have `sudo` on the host.
+- Your Executor relies on a pretrained model, you want to include this 100MB file into the image so that people don't need to download it again.  
 - You use Kubernetes or Docker Swarm and this orchestration framework requires each microservice to run as a Docker container.
 - You are using Jina on the cloud and you want to deploy an immutable Pod and version control it.
-- You have figured out a set of parameters that works best for an executor, you want to write it down in a YAML config and share it to others.
+- You have figured out a set of parameters that works best for an Executor, you want to write it down in a YAML config and share it to others.
 - You are debugging, doing try-and-error on exploring new packages, and you don't want ruin your local dev environments. 
 
 
-## What Should be in the Image?
+## What files should be in the Pod image?
 
-Typically, the following files are packed into the container image:
+Typically, the following files are packed into the container image.
 
 | File             | Descriptions                                                                                        |
 |------------------|-----------------------------------------------------------------------------------------------------|
 | `Dockerfile`     | describes the dependency setup and expose the entry point;                                          |
 | `build.args`     | metadata of the image, author, tags, etc. help the Hub to index and classify your image             |
-| `*.py`           | describes the executor logic written in Python, if applicable;                                      |
-| `*.yml`          | a YAML file describes the executor arguments and configs, if you want users to use your config;     |
-| Other data files | may be required to run the executor, e.g. pre-trained model, fine-tuned model, home-made data.      |
+| `*.py`           | describes the Executor logic written in Python, if applicable;                                      |
+| `*.yml`          | a YAML file describes the Executor arguments and configs, if you want users to use your config;     |
+| Other data files | may be required to run the Executor, e.g. pre-trained model, fine-tuned model, home-made data.      |
 
-Except `Dockerfile`, all others are optional to build a valid Pod image depending on your case. `build.args` is only required when you want to [upload your image to Jina Hub](./publish-your-pod-image.html#publish-your-pod-image-to-jina-hub).
+Except `Dockerfile`, all other options to build a valid Pod image depending on your case. `build.args` is only required when you want to [upload your image to Jina Hub](./publish-your-pod-image.html#publish-your-pod-image-to-jina-hub).
+
+## How to change the default drivers of the Executor that is running inside the Docker image?
```

Comment:
I miss an explanation on what this `uses_internal` do. 

It must be said that this `uses_internal` will override the `yaml` used inside the `pod image`.

Also you can set an example of the `config.yml` inside the image and an optional indexer.yml with different parameters and drivers